unit/src/NitrogenFluidPropertiesTest.C

```diff
@@ -0,0 +1,252 @@
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NitrogenFluidPropertiesTest.h"
+#include "SinglePhaseFluidPropertiesPTTestUtils.h"
+
+/**
+ * Test that the fluid name is correctly returned
+ */
+TEST_F(NitrogenFluidPropertiesTest, fluidName) { EXPECT_EQ(_fp->fluidName(), "nitrogen"); }
+
+/**
+ * Test that the molar mass is correctly returned
+ */
+TEST_F(NitrogenFluidPropertiesTest, molarMass)
+{
+  ABS_TEST(_fp->molarMass(), 28.01348e-3, REL_TOL_SAVED_VALUE);
+}
+
+/**
+ * Test that the critical properties are correctly returned
+ */
+TEST_F(NitrogenFluidPropertiesTest, criticalProperties)
+{
+  ABS_TEST(_fp->criticalPressure(), 3.3958e6, REL_TOL_SAVED_VALUE);
+  ABS_TEST(_fp->criticalTemperature(), 126.192, REL_TOL_SAVED_VALUE);
+  ABS_TEST(_fp->criticalDensity(), 313.3, REL_TOL_SAVED_VALUE);
+}
+
+/**
+ * Test that the triple point properties are correctly returned
+ */
+TEST_F(NitrogenFluidPropertiesTest, triplePointProperties)
+{
+  ABS_TEST(_fp->triplePointPressure(), 12.523e3, REL_TOL_SAVED_VALUE);
+  ABS_TEST(_fp->triplePointTemperature(), 63.151, REL_TOL_SAVED_VALUE);
+}
+
+/**
+ * Verify calculation of vapor pressure, vapor density and saturated liquid
+ * density
+ */
+TEST_F(NitrogenFluidPropertiesTest, vapor)
+{
+  const Real tol = REL_TOL_EXTERNAL_VALUE;
+
+  // Vapor pressure
+  REL_TEST(_fp->vaporPressure(80.0), 0.13687e6, tol);
+  REL_TEST(_fp->vaporPressure(100.0), 0.77827e6, tol);
+  REL_TEST(_fp->vaporPressure(120.0), 2.51058e6, tol);
+  //
+  // Saturated vapor density
+  REL_TEST(_fp->saturatedVaporDensity(80.0), 0.21737 * 1000.0 * _fp->molarMass(), tol);
+  REL_TEST(_fp->saturatedVaporDensity(100.0), 1.1409 * 1000.0 * _fp->molarMass(), tol);
+  REL_TEST(_fp->saturatedVaporDensity(120.0), 4.4653 * 1000.0 * _fp->molarMass(), tol);
+
+  // Saturated liquid density
+  REL_TEST(_fp->saturatedLiquidDensity(80.0), 28.341 * 1000.0 * _fp->molarMass(), tol);
+  REL_TEST(_fp->saturatedLiquidDensity(100.0), 24.608 * 1000.0 * _fp->molarMass(), tol);
+  REL_TEST(_fp->saturatedLiquidDensity(120.0), 18.682 * 1000.0 * _fp->molarMass(), tol);
+}
+
+/**
+ * Verify calculation of partial density at infinite dilution using data from
+ * Hnedkovsky et al., Volumes of aqueous solutions of CH4, Hydrogen, H2S, and NH3
+ * at temperatures from 298.15 K to 705 K and pressures to 35 MPa,
+ * J. Chem. Thermo. 28, 1996.
+ * As we are comparing with experimental data, calculated values within 5% are
+ * considered satisfactory.
+ */
+// TEST_F(NitrogenFluidPropertiesTest, partialDensity)
+// {
+//   const Real tol = 50.0 * REL_TOL_EXTERNAL_VALUE;
+//
+//   REL_TEST(_fp->partialDensity(373.15), 1182.8, tol);
+//   REL_TEST(_fp->partialDensity(473.35), 880.0, tol);
+//   REL_TEST(_fp->partialDensity(573.15), 593.8, tol);
+// }
```

Comment:
Was this supposed to be uncommented for the final version?

Reply:
Bad copy and paste. It shouldn't be here